### PR TITLE
CB-12018 : updated tests to work with jasmine instead of jasmine-node

### PR DESCRIPTION
**Pull request by audreyso**
CB-12018 : updated tests to work with jasmine instead of jasmine-node

### What does this PR do?
Uses jasmine to test instead of jasmine-node.

### Checklist
- [X] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [X] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [X] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by audreyso** [comment]
PR ready to be merged

File: package.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
   ],
   "scripts": {
     "test": "npm run jshint && npm run test-unit && npm run test-e2e",
-    "test-unit": "jasmine-node --captureExceptions spec/unit",
-    "test-e2e": "jasmine-node --captureExceptions spec/e2e",
-    "cover": "istanbul cover --root template --print detail node_modules/jasmine-node/bin/jasmine-node -- spec/unit",
+    "test-unit": "jasmine --captureExceptions ",
```
Comment (janpio) [inline package.json]: Unfortunately the option used here is not available in `jasmine` which caused these tests to not be run on Windows any more:

```
> cordova-windows@5.1.0-dev test-e2e C:\projects\cordova-windows
> jasmine --captureExceptions spec/e2e/endtoend.spec.js
Unknown options: --captureExceptions
Usage: jasmine [command] [options] [files]
Commands:
      init	initialize jasmine
  examples	install examples
   help,-h	show help
version,-v	show jasmine and jasmine-core versions
If no command is given, jasmine specs will be run
Options:
        --no-color	turn off color in spec output
         --filter=	filter specs to run only those that match the given string
         --helper=	load helper files that match the given string
--stop-on-failure=	[true|false] stop spec execution on expectation failure
         --config=	path to your optional jasmine.json
The given arguments take precedence over options in your jasmine.json
The path to your optional jasmine.json can also be configured by setting the JASMINE_CONFIG_PATH environment variable
Build success
```

Example: https://ci.appveyor.com/project/ApacheSoftwareFoundation/cordova-windows/build/1.0.404/job/61k4u344jus9t15l

Comment (janpio) [reply package.json]: Turns out back when this was commited it probably did work, only later `jasmine` versions removed it.

**Comment by brodybits** [comment]
Adding link to original issue raised at <https://issues.apache.org/jira/browse/CB-12018>

**Pull request merged**